<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolMainTypeDialog</class>
 <widget class="QDialog" name="ToolMainTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="toolMainTypeLabel">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>650</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 144, 144);</string>
   </property>
   <property name="text">
    <string>工具分类</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="mainTipsTextBrowser">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>350</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;提示&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:12pt;&quot;&gt;      点击工具时显示对应的说明信息！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertToolBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>690</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </widget>
  <widget class="QWidget" name="toolSubTypeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>690</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>690</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addToolBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>690</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>追加</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="detailTipsTextBrowser">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>318</y>
     <width>350</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;提示&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑'; font-size:12pt;&quot;&gt;      点击工具时显示对应的说明信息！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetType">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>30</y>
     <width>650</width>
     <height>291</height>
    </rect>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LabelDetail">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>325</y>
     <width>650</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 144, 144);</string>
   </property>
   <property name="text">
    <string>检测工具选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetDetail">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>367</y>
     <width>650</width>
     <height>310</height>
    </rect>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
